lib/features/base/reloadable/reloadable_controller.dart

```diff
@@ -157,7 +158,9 @@ abstract class ReloadableController extends BaseController {
       ]);
       _authorizationInterceptors.clear();
       _authorizationIsolateInterceptors.clear();
+      FcmController.instance.deleteFcmToken();
       await HiveCacheConfig().closeHive();
```

Comment:
@dab246 How about move it also to Caching Manager. It is not good if this controller depends on Hive, Firebase or other third-party lib